src/lib/core/hooks/findOutputEntity.ts

```diff
@@ -8,7 +8,7 @@ export function useFindOutputEntity(
   // need to add string at Record's Type due to valueSpecConfig
   dataElementVariables: Record<
     string,
-    VariableDescriptor | string | boolean | undefined
+    VariableDescriptor | string | boolean | string[] | undefined
```

Comment:
Oh, I see why... because we pass the whole of `vizConfig` to this function.  Looks like some technical debt here!

Reply:
yes indeed